app/controllers/users_controller.rb

```diff
@@ -1,3 +1,28 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find_by id: params[:id]
+    if @user.blank?
```

Comment:
chỗ này nên viết cách khác, a chạy rubocop nó có suggest cách fix đó
nếu k đc a TO e hỉ